src/components/paragraph/stanford-entity.tsx

```diff
@@ -60,15 +60,24 @@ const StanfordEntity = async ({
           <div className="mb-40">{formatHtml(description)}</div>
         }
 
-        {entities &&
-          <div className={`mb-40 grid gap-[90px] ${gridClass}`} aria-live="polite">
-            {entities.map(item =>
-              <div key={item.id} className="mx-auto w-full">
-                <NodeCardDisplay node={item} h3Heading={!!headline}/>
-              </div>
-            )}
-          </div>
+        {entities?.length > 1 ? (
+          <ul className={`list-unstyled mb-40 grid gap-[90px] ${gridClass}`} aria-live="polite">
+            {entities.map((item, index) => ( 
+              <li key={item.id} className="mx-auto w-full">
+                <NodeCardDisplay node={item} h3Heading={headline}/>
+              </li>)
+            )
+            }
+          </ul>
+          )
+          :
+          (
+            <div className="mx-auto w-full">
+              <NodeCardDisplay node={entities[0]} h3Heading={headline}/>
+            </div>
+          )
```

Comment:
@pookmish I changed it to this so the single entity is a `div` and the more than one is a list.  What do you think of this route?